src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -2012,10 +2012,10 @@ public async Task<ConnectionState> WaitForActiveConnectionAsync(string methodNam
                 return CurrentConnectionStateUnsynchronized;
             }
 
-            public void ReleaseConnectionLock([CallerMemberName] string memberName = null,
-                [CallerFilePath] string filePath = null, [CallerLineNumber] int lineNumber = 0)
+            public void ReleaseConnectionLock([CallerMemberName] string? memberName = null,
+                [CallerFilePath] string? filePath = null, [CallerLineNumber] int lineNumber = 0)
             {
-                Log.ReleasingConnectionLock(_logger, memberName, filePath, lineNumber);
+                Log.ReleasingConnectionLock(_logger, memberName!, filePath!, lineNumber);
```

Comment:
It might better to just make these `string?` in `LoggerMessage.Define` to avoid `!` fatigue (where `!` is so common we stop paying much attention to them).